apache/superset
# feat: Environment tag

### SUMMARY
It can often be useful to see at a glance what environment you're currently in. This configurable tag can be set to appear based on any environment variable you set. By default, it shows a tag for development or production based on FLASK_ENV.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before
![image](https://user-images.githubusercontent.com/10563996/172448071-d569c9e9-3537-4b08-889a-23a0ea470b2d.png)
After
![image](https://user-images.githubusercontent.com/10563996/172448006-225b0de7-3f2d-4e4e-8d38-7097bd18bcc4.png)

### TESTING INSTRUCTIONS
1. Start superset in production mode. The tag should not appear
2. Start superset in development mode. The tag should appear and say "Development"

### ADDITIONAL INFORMATION
